addons/website_twitter/views/res_config_settings_views.xml

```diff
@@ -30,21 +30,17 @@
                                 <blockquote class="small">
                                     <h2>How to configure the Twitter API access</h2>
                                     <ol>
-                                        <li>Create a new Twitter application on <a href="https://apps.twitter.com/app/new" target="new">https://apps.twitter.com/app/new</a>
-                                              with the following values:
+                                        <li>Log in or create an account on <a href="https://developer.twitter.com/" target="new"> https://developer.twitter.com/</a></li>
+                                        <li>Once connected, go to <a href="https://developer.twitter.com/portal/" target="new">https://developer.twitter.com/portal/</a></li>
+                                        <li>Complete the Twitter portal access process and <strong>create a project</strong> with the following information:
                                             <ul>
                                                 <li><strong>Name: </strong> Odoo Twitter Integration</li>
+                                                <li><strong>Use Case: </strong> Embedding Tweets in a website</li>
```

Comment:
Are you sure about all this? The only field I can see is the app name (the first step is just about choosing Dev/Staging/prod)
![image](https://user-images.githubusercontent.com/30048408/170963825-ccb77b8e-1d82-4864-9926-da15350385f8.png)